setup.py

```diff
@@ -23,10 +23,10 @@
 
 setup(
     name='gcloud',
-    version='0.4.3',
+    version='0.5.0',
     description='API Client library for Google Cloud',
-    author='JJ Geewax',
-    author_email='jj@geewax.org',
+    author='FOO BAR',
+    author_email='foo@bar.org',
```

Comment:
Maybe the author should read 'Google Cloud Developers', with an e-mail address which points to a mailing list?